driver/InputFlags.cpp

```diff
@@ -27,21 +27,25 @@
 #include <iomanip>
 #include <iostream>
 #include <vector>
+#include <algorithm>
 
 InputFlags::InputFlags() { AddInputFlag("help", 'h', "", "Print Help Message", "string"); }
 
 void InputFlags::AddInputFlag(const std::string& _long_name,
                               char _short_name,
                               const std::string& _value,
                               const std::string& _help_text,
-                              const std::string& _type)
+                              const std::string& _type,
+                              const bool _is_case_insensitive)
```

Comment:
[Quality] `is_case_insensitive` -> `convert2uppercase` or something alike. @DrizztDoUrden suggests that this more clearly explains what really happens and I agree wit him. We do not really use case-insensitive compare but convert input options instead.

Reply:
variable name is changed to convert2uppercase